adapter/CCXHelpers.h

```diff
@@ -306,7 +306,32 @@ bool isEqual(const char * lhs, const char * rhs);
  * @param a is one double in the comparison
  * @param b is the other double in the comparison
  */
-bool isDoubleEqual(double a, double b);
+bool isDoubleEqual(const double a, const double b);
+
+/**
+ * @brief Set all values of an array to 0
+ * @param values is the array carrying double values
+ * @param length is the number of elements in array
+ */
+void setDoubleArrayZero(double * values, const int length);
+
+/**
+ * @brief Maps data from 2D mesh nodes to 3D mesh nodes
```

Comment:
```suggestion
 * @brief Maps vector-valued data from 2D mesh nodes to 3D mesh nodes
```
The CalculiX adapter also supports CHT, where we also couple scalar data (temperature). We should keep this in mind for later.